### ISSUE TITLE
Execution context

### ISSUE THREAD

**Issue by crisu83** [digiaonline/graphql-php#4]

**Comment by crisu83** [comment]
Re-opened, because @hungneox is still working on this.

**Comment by crisu83** [comment]
@hungneox can we consider this issue done after #40? We could open another issue for the lazy resolving of fields.

**Comment by crisu83** [comment]
@hungneox is this issue done?

**Comment by crisu83** [comment]
This has been done long ago.
